### DISCUSSION TITLE
Verified but no access to features on renewed account

### DISCUSSION THREAD

**Discussion by Peter-K-Joseph** [community/community#84950] [GitHub Education]
### Hi everyone,

Hey everyone,
So I am currently verified and all but i still dont have access to the features for a couple of days now
It shows this when I try to reverify
![image](https://github.com/community/community/assets/69970683/a5145340-96a0-4f23-bbe7-4fdcbfd2deb0)

But now my github shows this
![image](https://github.com/community/community/assets/69970683/097ef67f-d235-41d2-a696-a6a4ee03092c)

How much longer does it take for the account to get activated. A week or something? I couldn't find a direct point of support for the same. Thanks

**Comment by KorigamiK** [comment]
same man, i hope it happens sooner
